classifier/cut_bbox_litter.py

```diff
@@ -0,0 +1,148 @@
+'''
+Script to prepare images for litter classification.
+'''
+import os
+import json
+import cv2
+import argparse
+
+
+def get_args_parser():
+    parser = argparse.ArgumentParser(
+        'Prepare images of trash for classification task')
+    parser.add_argument('--src_coco', help='path to coco annotations',
+                        type=str)
+    parser.add_argument('--src_img',
+                        help='path to source directory with images',
+                        type=str)
+    parser.add_argument('--dst_img',
+                        help='path to destination directory for images',
+                        type=str)
+    parser.add_argument('--name',
+                        help='type of annotations: wimlds or epi',
+                        default='wimlds',
+                        choices=['wimlds', 'epi'],
+                        type=str)
+    return parser
+
+
+epi_map = {1: 4,
```

Comment:
consider using  annotations_preprocessing.py from main repo instead of the new one for easier version managment

Reply:
At the time of writing this script we had no data from epinote yet, and data split has yet to be done. And that why I decided to work on raw annotations from Pedro and then epinote. But yes, I agree in next step we should use the same annotation file. But this scrip allows us to prepare some data for classification task, but there is some problem with annotations - negative values for x and/or y position of bbox... 
This usually happens at the edge of the image, and I took abs od this values, and I know it is not the best option...